.air.toml

```diff
@@ -0,0 +1,55 @@
+# Config file for SpinUp in TOML format
+
+# Working directory
+# . or absolute path, please note that the directories following must be under root.
+root = "."
+tmp_dir = "./tmp"
+
+[build]
+# Just plain old shell command. You could use `make` as well.
+cmd = "go build -o ./tmp/main ."
+# Binary file yields from `cmd`.
+bin = "./tmp/main"
+# Customize binary, can setup environment variables when run your app.
+full_bin = "APP_ENV=dev APP_USER=air ./tmp/main"
+# Watch these filename extensions.
+include_ext = ["go", "tpl", "tmpl", "html"]
+# Ignore these filename extensions or directories.
+exclude_dir = ["assets", "tmp", "vendor", "frontend/node_modules"]
```

Comment:
was this copied from some other project? I'm not sure we use any of these directories at the moment. Maybe we can tidy the config to clean up stuff we don't need?

Reply:
yes, i copied it from standard air project, kept things there just so that we can use it if req in near future.

Reply:
makefile has been added in the Gitignore someone, dint notice it, so just pushed it.